src/libs/actions/Policy/Category.ts

```diff
@@ -542,8 +542,11 @@ function renamePolicyCategory(policyID: string, policyCategory: {oldName: string
     const policyCategoryExpenseRule = CategoryUtils.getCategoryExpenseRule(policy?.rules?.expenseRules ?? [], policyCategory.oldName);
     const approvalRules = policy?.rules?.approvalRules ?? [];
     const expenseRules = policy?.rules?.expenseRules ?? [];
+    const mccGroup = policy?.mccGroup ?? [];
```

Comment:
```suggestion
    const mccGroup = policy?.mccGroup ?? {};
```

Reply:
This is the cause for `lodashCloneDeep` inferring the never type, mccGroup is an object